agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -394,6 +394,45 @@ bool slave_thread::handle_cmdu_control_ieee1905_1_message(Socket *sd,
     return true;
 }
 
+bool slave_thread::start_monitoring(const beerocks_message::sClientMonitoringParams &params,
+                                    std::shared_ptr<beerocks_header> beerocks_header)
```

Comment:
It is better to pass `beerocks_header` as reference. It is also in our [CODINGSTYLE.md](https://github.com/prplfoundation/prplMesh/blob/master/CODINGSTYLE.md).